docs/excel/excel-add-ins-advanced-concepts.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Excel JavaScript API advanced concepts
+title: Advanced Excel add-in programming
 description: ''
-ms.date: 1/18/2018
+ms.date: 10/03/2018
 ---
 
 
 
-# Excel JavaScript API advanced concepts
+# Advanced Excel JavaScript add-in programming
 
 This article builds upon the information in [Excel JavaScript API core concepts](excel-add-ins-core-concepts.md) to describe some of the more advanced concepts that are essential to building complex add-ins for Excel 2016 or later.
```

Comment:
Link text should be updated to new article title "Beginner … " (But see my other comment about that title.)